src/test.js

```diff
@@ -374,7 +374,9 @@ Test.prototype = {
 				resume = internalStop( test );
 				then.call(
 					promise,
-					function() { resume(); },
+					function() {
+						resume();
+					},
```

Comment:
I'll look into this-- that might have just been me being OCD. (But I'm not 100% sure.)